data/sdss5.cfg

```diff
@@ -221,3 +221,6 @@ asR = $APOGEE_DATA_2S/{mjd}/asR-{chip}-{num:0>8}.apz
 aspcapField = $APOGEE_ASPCAP/{apred}/{aspcap}/{telescope}/{field}/aspcapField-{field}.fits
 aspcapStar = $APOGEE_ASPCAP/{apred}/{aspcap}/{telescope}/{field}/aspcapStar-{apred}-{obj}.fits
 aspcapStar-1m = $APOGEE_ASPCAP/{apred}/{aspcap}/{telescope}/{field}/aspcapStar-{apred}-{reduction}.fits
+
+# boss paths
+spSpec = $BOSS_SPECTRO_REDUX/{run2d}/{plate}p/coadd/{mjd}/spSpec-{plate}-{mjd}-{max(fiberid, catalogid):0>11}.fits
```

Comment:
The paths for `tree` and `sdss_access` don't support complex operations or functions for the variable substitution syntax.  We're not currently using f-strings, just a simple string substitution via `.format`.  I don't think `{max(fiberid, catalogid)}` will work in that mode.  Can you simplify those keyword variables?  I would recommend making these path definitions as simple as possible.

Alternatively if you really want to take the max of either variable, you could define a special function using the `@[name()]|` syntax, e.g. the `apField` entry.  Then define that function in `sdss_access.path.Path`.